analytics/src/main/webapp/WEB-INF/ws-servlet.xml

```diff
@@ -53,6 +53,8 @@
                 <entry key="headerHeight" value="${headerHeight}"/>
                 <entry key="headerScript" value="${headerScript}" />
                 <entry key="useLegacyHeader" value="${useLegacyHeader}" />
+                <entry key="logoUrl" value="${logoUrl}" />
+                <entry key="stylesheet" value="${georchestraStylesheet}" />
```

Comment:
The key should also be `georchestraStylesheet` I think. Noticed it when testing. It's a pretty long name, btw, but I guess it makes sense to be explicit.